languages/python.md

```diff
@@ -23,29 +36,40 @@ def widgetize(widget):
   standardize(widget)
 ```
 
-## Linting
-It is recommended to use [black](https://black.readthedocs.io/en/stable/) (formatting), [flake8](https://flake8.pycqa.org/en/latest/index.html#quickstart) (formatting), [bandit](https://bandit.readthedocs.io/en/latest/) (security),[mypy](https://mypy.readthedocs.io/en/stable/) (type hinting), and [isort](https://pycqa.github.io/isort/) (import order) during development and CI. Black is preferred as the primary formatting style due to its easy methods for auto-formatting the code. 
-
-To install these linters when starting a new project, run `pipenv install --dev black flake8 bandit mypy isort`. The linters can be run together by adding a `make lint` command to the project's Makefile. See the [Makefile](https://github.com/MITLibraries/alma-scripts/blob/main/Makefile) for the `llama` application for an example.
+### Linting, code quality, and code safety
 
+Our standard Python code checkers in all repositories include:
 
+- [bandit](https://bandit.readthedocs.io/en/latest/) - security
+- [black](https://black.readthedocs.io/en/stable/) - formatting
+- [isort](https://pycqa.github.io/isort/) - import order
+- [mypy](https://mypy.readthedocs.io/en/stable/) - type hinting
+- [pylama](https://github.com/klen/pylama) - formatting and code quality (wrapper around several other tools)
 
+These tools should be used during development and are run automatically in Github Actions during CI. They are all included in the template repositories listed above, and have integrations for common code editors to allow automatic checking and reformatting during development.
 
-## Command line interfaces
-Typically, [click](https://click.palletsprojects.com/en/8.0.x/) is used for creating command line interfaces (CLI) in Python projects.
+The linters are usually run together with the `make lint` command in a project's Makefile. See the template repositories for examples.
 
 ## Dependencies
 
-Use [Pipenv](https://pipenv.readthedocs.io/en/latest/) to manage dependencies. If there's some reason you need to support pip, then you should still manage dependencies with pipenv, but generate your `requirements.txt` file as needed with:
+Use [Pipenv](https://pipenv.readthedocs.io/en/latest/) to manage dependencies for Python applications. If there's some reason you need to support pip, then you should still manage dependencies with pipenv, but generate your `requirements.txt` file as needed with:
```

Comment:
Would an example of where `pip` would be required at runtime be building lambdas using the aws base image?

Reply:
Yeah. That's demo'd in the template repo.